CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -2055,10 +2055,6 @@ public static function submit($params) {
     $form->controller = new CRM_Contribute_Controller_Contribution();
```

Comment:
This function is only called by ContributionPage.submit api which is either mostly or only used by unit tests